src/util/session.js

```diff
@@ -72,6 +72,22 @@ import { forceReplace } from './router';
 import { localStore } from './storage';
 import { noop } from './util';
 
+const removeSessionFromStorage = () => {
```

Comment:
This is copied from below. It's in its own function now that it is used in a couple of different places.